src/live_restore/live_restore_fs.c

```diff
@@ -896,11 +896,13 @@ __live_restore_fh_close(WT_FILE_HANDLE *fh, WT_SESSION *wt_session)
     session = (WT_SESSION_IMPL *)wt_session;
     __wt_verbose_debug1(session, WT_VERB_FILEOPS, "LIVE_RESTORE_FS: Closing file: %s\n", fh->name);
 
-    if (FLD_ISSET(
-          lr_fh->destination.back_pointer->debug_flags, WT_LIVE_RESTORE_DEBUG_FILL_HOLES_ON_CLOSE))
-        WT_RET(__wti_live_restore_fs_fill_holes(fh, wt_session));
+    if (lr_fh->destination.fh != NULL) {
```

Comment:
Can we add a comment explaining when it reasonable for not having a destination file handle? I am a bit confused by this myself.

Reply:
[1fe24c5](https://github.com/wiredtiger/wiredtiger/pull/11379/commits/1fe24c5407d375cef1de5a0536e4ee4aa57eee51)